portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchFacets.testcase

```diff
@@ -780,7 +780,7 @@ definition {
 
 	@priority = 4
 	test AssertNonAdminsCannotAddCertainWidgets {
-		property custom.properties = "feature.flag.LPS-155692=true";
+		property custom.properties = "layout.user.plid.access.denied=true";
```

Comment:
Shouldn't we set the property to false here? Enabling the FF means allowing user pages to be accessed by plid.